page_objects/accAfterReg_page.js

```diff
@@ -0,0 +1,15 @@
+let BasePage = require("../base/base.page")
+let TextView = require("../elements/text.view")
+
+let successfulReg = by.css('.info-account')
+
+class Registration extends BasePage {
```

Comment:
The class name already used in other page object

Reply:
fixed